vuepress/docs/v7.0/docs/consume/entandoapp-cr.md

```diff
@@ -0,0 +1,39 @@
+ 
+Entando offers standard Wildfly or Eap images for the definition, but typically customers provide their own.
+
+## Example EntandoApp
+ 
+```yaml
+apiVersion: entando.org/v1
+kind: EntandoApp
+metadata:
+  namespace: entando
+  name: my-app    
+spec:
+  environmentVariables: []
+  dbms: embedded
+  ingressHostName: my-app.192.168.64.5.nip.io
+  standardServerImage: eap
+  replicas: 1
+ ```
+## Specifications 
+| Spec Name | Description |
+| :- | :- |
+| `spec.customServerImage`|  Used to deploy the Docker image containing your custom Entando App. Follow these instructions on how to [build your own image](../../tutorials/devops/build-core-image.md). This property and the `spec.standardServerImage` are  mutually exclusive.|
+|`spec.dbms` | Allowed values are: MySQL, PostgreSQL (default), Oracle, or embedded. Oracle is only supported as an external database.|
+|`spec.ecrGitSshSecretName`| The configuration used by the Entando Component Repository to download bundles from authenticated git repositories. It's a secret containing a private key file named `rsa_id` that matches a public key configured in the authenticated Git repository.|
+|`spec.environmentVariables`| A map of environment variables to pass to the EntandoApp Docker image. This can be used to provide connection details for custom datasources or message queues as discussed in the [custom datasources tutorial](../../tutorials/devops/change-default-datasource.md). These variables can sometimes be used as a mechanism to override any of the default environment variables that need customization.|
+|`spec.ingressPath`| Specifies the web context of the EntandoApp to be deployed. |
+|`spec.ingressHostName`| The hostname of the Kubernetes ingress to be created for the EntandoApp.EntandoPlugins linked to this app will also be made available on this host.|
+|`spec.replicas`| The number of replicas to be made available on the deployment.|
+|`spec.resourceRequirements`| The minimum and maximum [resource allocation](./custom-resources.md#general-resourcerequirements-specifications) for the Entando App Engine container|
+|`spec.serviceAccountToUse`| The Kubernetes ServiceAccount in the namespace of the EntandoApp that is to be used for the pods hosting the EntandoApps. The default is 'default'.|
+|`spec.standardServerImage`| Either a `wildfly` or `eap` image. This property and the `spec.customServerImage` are mutually exclusive. Refer to the  [Docker image section](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/charts/entando-k8s-controller-coordinator/README.md#how-it-resolves-docker-images) to determine how the Docker registry and versions are calculated.|
+|`spec.storageClass` | Name of the StorageClass to use for PersistentVolumeClaims created for this EntandoApp. For more information, go to [Kubernetes explanation on Storage Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/).|
```

Comment:
...Classes]. (add period)

Reply:
there is after the URL

Reply:
not sure what i was looking at here? the . is very obv there...